frame/contracts/src/benchmarking/mod.rs

```diff
@@ -534,6 +534,38 @@ benchmarks! {
 		let origin = RawOrigin::Signed(instance.caller.clone());
 	}: call(origin, instance.addr, 0u32.into(), Weight::MAX, None, vec![])
 
+	#[pov_mode = Measured]
+	seal_caller_is_root {
```

Comment:
@athei @agryaznov I am not too familiarised with benchmarking, so not sure if this is the right approach. This is pretty much a copy-pasta from `seal_caller_is_origin` with a few adaptations

Reply:
This can be simplified by using `WasmModule::getter`. Look at the `seal_address` benchmark.

Reply:
Yes you are right. Getters copy to memory not return on stack. Forgot about that. So you need to roll this back.